src/devices/tuya.ts

```diff
@@ -5029,6 +5029,60 @@ const definitions: Definition[] = [
             ],
         },
     },
+    {
+        fingerprint: [
+            {
+                modelID: 'TS0601',
+                manufacturerName: '_TZE200_0j5jma9b',
+            },
+        ],
+        model: 'G3 Smart Switch',
+        vendor: 'TuYa',
+        description: 'TuYa Smart Switch (4 gang + 2 scene) with neutral wire',
+        fromZigbee: [tuya.fz.datapoints],
+        toZigbee: [tuya.tz.datapoints],
+        configure: tuya.configureMagicPacket,
+        exposes: [
+            tuya.exposes.switch().withEndpoint('l1'),
+            tuya.exposes.switch().withEndpoint('l2'),
+            tuya.exposes.switch().withEndpoint('l3'),
+            tuya.exposes.switch().withEndpoint('l4'),
+            tuya.exposes.switch().withEndpoint('l5'),
+            tuya.exposes.switch().withEndpoint('l6'),
+            tuya.exposes.backlightModeOffOn(),
+            tuya.exposes.countdown().withEndpoint('l1'),
+            tuya.exposes.countdown().withEndpoint('l2'),
+            tuya.exposes.countdown().withEndpoint('l3'),
+            tuya.exposes.countdown().withEndpoint('l4'),
+            tuya.exposes.countdown().withEndpoint('l5'),
+            tuya.exposes.countdown().withEndpoint('l6'),
+            tuya.exposes.switch().withEndpoint('master_switch'),
```

Comment:
What happens if you remove `tuya.exposes.switch().withEndpoint('master_switch'),` here and change `[13, 'state_master', tuya.valueConverter.onOff],` to `[13, 'state', tuya.valueConverter.onOff],`, does this work?

Reply:
Removing `tuya.exposes.switch().withEndpoint('master_switch'),` no longer exposes the master switch in the frontend.
However, publishing {"state": "ON"} and {"state": "OFF"} does the trick.

Reply:
Sorry, I meant changing ``tuya.exposes.switch().withEndpoint('master_switch')` to ``tuya.exposes.switch()`

Reply:
Changing to `tuya.exposes.switch()` works as properly.
But I can't seem to add an discription with `tuya.exposes.switch().withDescription('All Switches'),`